recipe/bld.bat

```diff
@@ -1,6 +1,16 @@
 mkdir build
 cd build
 
+:: Configure using the CMakeFiles
+
+set CTEST_EXCLUDES="test_priors"
+:: exclude tests using parallelproj GPU
+if NOT "%cuda_compiler_version%"=="None" (
+  set CTEST_EXCLUDES="%CTEST_EXCLUDES%|parallelproj|test_blocks_on_cylindrical_projectors"
```

Comment:
I don't think that `|` is valid syntax in a batch script.

Reply:
sorry for my lack of batch script knowledge.
Should it be:
```
  set CTEST_EXCLUDES="(%CTEST_EXCLUDES%|parallelproj|test_blocks_on_cylindrical_projectors)"
```
instead with extra `()`

Reply:
or does anyone know what the correct syntax for `ctest -E <a list of tests>` in batch is?

Reply:
the problem was in the `echo` statement. It needed some extra quotes apparently. Should be fixed now.